website/docs/docs/core/pip-install.md

```diff
@@ -79,23 +79,23 @@ You can upgrade or downgrade versions of dbt Core by using the `--upgrade` optio
 To upgrade dbt to the latest version:
 
 ```
-pip install --upgrade dbt-core
+python -m pip install --upgrade dbt-core
 ```
 
 To downgrade to an older version, specify the version you want to use. This command can be useful when you're resolving package dependencies. As an example:
 
 ```
-pip install --upgrade dbt-core==0.19.0
+python -m pip install --upgrade dbt-core==0.19.0
 ```
 
-### `pip install dbt`
+### `python -m pip install dbt`
 
 Note that, as of v1.0.0, `pip install dbt` is no longer supported and will raise an explicit error. Since v0.13, the PyPI package named `dbt` was a simple "pass-through" of `dbt-core` and the four original database adapter plugins. For v1, we formalized that split.
```

Comment:
[`dbt` is back!](https://docs.getdbt.com/docs/cloud/cloud-cli-installation?install=pip#install-dbt-cloud-cli-in-pip)

We should open a separate issue to update this blurb to mention the timeline:
- Since v0.13, PyPI package named `dbt` was a simple "pass-through" of `dbt-core` and the four original database adapter plugins
- Upon the release of dbt-core v1.0.0, `pip install dbt` was no longer a supported installation method by dbt Labs, and the `dbt` package on PyPI stopped receiving updates
- In fall of 2023,  the `dbt` package on PyPI came back to life as a way to install the [dbt Cloud CLI](https://docs.getdbt.com/docs/cloud/cloud-cli-installation?install=pip#install-dbt-cloud-cli-in-pip)

Reply:
Captured this here https://github.com/dbt-labs/docs.getdbt.com/issues/4515